lib/Pipfile

```diff
@@ -41,7 +41,7 @@ numpy = "*"
 packaging = "*"
 pandas = ">=0.21.0"
 pillow = ">=6.2.0"
-protobuf = ">=3.6.0"
+protobuf = ">=3.13.0"
```

Comment:
Thanks for submitting this change, @darshkpatel! Upon discussion with the team, we're hesitant to require such a huge jump in protobuf version; it could break other dependencies (your screenshot shows mitmproxy not expecting 3.13.0 haha)

Could we instead just forbid the bad version of protobuf (3.11, I believe)? And add a comment so we know why:

```
# protobuf version 3.11 is incompatible, see https://github.com/streamlit/streamlit/issues/2234
protobuf = ">=3.6.0, !=3.11"
```